index.js

```diff
@@ -7,8 +7,27 @@ var uuid = require("uuid");
 var nodemailer = require("nodemailer");
 var faker = require("faker");
 var profileModel = require(__dirname + "/../../api/profiles/index").model();
-var inMemoryTokens = {};
-var inMemoryKeys = {};
+
+var inMemory = {
```

Comment:
Put this into a separate file, and use this pattern

```
var InMemory = function() {
   if (InMemory.prototype._singleton) {
      return InMemory.prototype._singleton;
   }
   InMemory.prototype._singleton = this;
   this.tokens = {};
   this.keys = {};
}

InMemory.prototype.set = function(token) {
  // ...
}

InMemory.prototype.unset = function(token) {
  // ...
}
```

You can check this is really a singleton, eg. by running this in REPL:
```
var x = new InMemory();
var b = InMemory();

x === b;  // it should print true
```